course_content/notebooks/numpy_intro.ipynb

```diff
@@ -1527,7 +1509,6 @@
     "### Functions:\n",
     "\n",
     "  * [empty](https://docs.scipy.org/doc/numpy/reference/generated/numpy.empty.html)\n",
-    "  * [meshgrid](https://docs.scipy.org/doc/numpy/reference/generated/numpy.meshgrid.html)\n",
```

Comment:
Why was this removed?

Reply:
It's [now covered in the main course content](https://render.githubusercontent.com/view/ipynb?commit=5c2999f0a41c2e05f3c53ebe99c70fa1244e0dad&enc_url=68747470733a2f2f7261772e67697468756275736572636f6e74656e742e636f6d2f646b696c6c69636b2f636f75727365732f356332393939663061343163326530356633633533656265393963373066613132343465306461642f636f757273655f636f6e74656e742f6e6f7465626f6f6b732f6e756d70795f696e74726f2e6970796e62&nwo=dkillick%2Fcourses&path=course_content%2Fnotebooks%2Fnumpy_intro.ipynb&repository_id=39770343&repository_type=Repository#Generating-2D-coordinate-arrays). There's no point in also having it in further reading and it's sort of misleading to do so as the further reading section is supposed to contain links to content _not_ covered by the rest of the course!